Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,40 @@
+//1. Declare a variable on each type using two options: plain creation, creation with constructor.1
+let a = 123;
+a = new Number(123);
+a = 'Hello';
+a = new String('Hello');
+a = true;
+a = new Boolean(true);
+a = null;
+a = undefined;
+a = Symbol('Hi');
+a = {
+  1: 'First',
+  2: 'Second'
+};
+
+// 2.Use operator typeof on each of them.
+let a = typeof (123);
+a = typeof ('Hello');
```

Comment:
you need to run it on variables that you should create above, not just reassigning `a`